pimd/pim6_cmd.c

```diff
@@ -42,7 +42,94 @@
 #include "pimd/pim6_cmd_clippy.c"
 #endif
 
+DEFPY (ipv6_pim_rp,
```

Comment:
Looks , documentation update missed for these new CLIs , please take care.

Reply:
@rgirada: Document updation is done in separate PR https://github.com/FRRouting/frr/pull/10480. I have mentioned about it in the description.